<?xml version="1.0" encoding="UTF-8"?>
<launch>

    <arg name="headless" />
    <arg name="output" default="log" />
    <arg name="rviz_config_file" />

    <node name="rviz"
          pkg="rviz"
          type="rviz"
          args="-d $(arg rviz_config_file)"
          unless="$(arg headless)"
          output="$(arg output)" />

</launch>
